packages/web/src/ssr/util.ts

```diff
@@ -0,0 +1,38 @@
+import { resolveRoute } from 'vike/routing'
```

Comment:
love this was broken out, can we rename this file to "makePageRoute"?

Reply:
i added the SSR audiusSdk instance to this file. not sure how we feel about toolbox util files so I can split it up if we hate it

Reply:
yeah as we add stuff lets eventually make a utils directory and have each piece of functionality in separate file?